src/components/Card.js

```diff
@@ -1,26 +1,62 @@
 import styled from "styled-components/macro";
+import { useState } from "react";
+
+function ActivityCard({
+  sport,
+  name,
+  route,
+  date,
+  time,
+  distance,
+  duration,
+  postalcode,
+  city,
+  street,
+  info,
+}) {
+  const [showDetails, setshowDetails] = useState(false);
+
+  const handleDetailsButtonClick = () => {
+    setshowDetails(showDetails ? false : true);
```

Comment:
I would use the `!` (NOT) operator. 
```suggestion
    setshowDetails(!showDetails);
```

Reply:
Already done, thx!